### ISSUE TITLE
Loot Pet not dropping certain ores

### ISSUE THREAD

**Issue by kyouma1312** [capnkirok/Inventory-Pets#258]
With the loot pet activated, it won't drop Coal, Diamonds or lapis lazuli. It works with everything else that i have tried. 

Thanks in advanced

**Comment by Emilgardis** [comment]
I'm having the same issue.

Mods used: [link to gist](https://gist.github.com/Emilgardis/3ad33e4a72fa54a3cb85596ce17c3fc0)

### Observations
* Using any pickaxe (including pickaxes with fortune) on Coal, Lapis, Diamond, Emerald or Nether Quartz ore with the loot pet produces no ore/drops.
* First use of Fortune with looting pet seems to make the ore drop items. Further use (after message about incompatibility with fortune etc enchants) with pet does not drop any ore.
* Redstone ore drops more with pet (as expected)

**Comment by kyouma1312** [comment]
yea pretty much same exact issue. im gonna try messing around a bit to try to fix it

**Comment by Darkmega18** [comment]
I'm not using any kind of fortune on the current pick (or my hands which mine at iron level thanks to monk mod). But for me, forestry's versions of metal ores are not dropping two ingots like the loot pet seems to do. also as described, coal, lapis and diamond, haven't tried emerald are all bugging right out.

 a lot of modded ores are working tho, like matter overdrive dilithium and tritanium, and mystical aggriculture infernium essence and prosperity, even minestuck cruxite and uranium lumps. But kinda salty that it's not working on the droppables for basic game.,, but I just cheated the ore back in and mined it normally then since I'm using ore excavator, which does proc properly for the stuff that actually works.

Here I was thinking I had found a wonderful little guy I barely end up using normally, until he started eating all the good stuff. >_>
